leaflet-treering.js

```diff
@@ -2743,6 +2806,173 @@ function ImageAdjustment(Lt) {
 
 }
 
+/**
+* Change measurement options (set subAnnual, previously hasLatewood, and direction)
+* @constructor
+* @param {Ltreeing} Lt - Leaflet treering object
+*/
+function MeasurementOptions(Lt) {
+  this.choice = false;
+  this.btn = new Button(
+    'timeline',
+    'Change measurement direction and annual/sub-annual mode',
+    () => { Lt.disableTools(); this.enable() },
+    () => { this.disable() }
+  );
+
+  /**
+  * Data from Lt.preferences
+  * @function preferencesInfo
+  */
+  MeasurementOptions.prototype.preferencesInfo = function () {
+    if (Lt.preferences.forwardDirection == false) { // direction object
+      this.forwardDirection = false;
+    } else {
+      this.forwardDirection = true;
+    }
+
+    let ewFalse = [];
+    for (i = 0; i <= Lt.data.points.length; i++) { // for assets which use hasLatewood
+       if (Lt.data.points[i] && Lt.data.points[i].earlywood == false) {
+         ewFalse.push(i);
+       }};
+    if (ewFalse.length > 0) {
+      this.hasLatewood = true;
+    } else {
+      this.hasLatewood = false;
+    };
+
+    this.subAnnual = Lt.preferences.subAnnual || this.hasLatewood || false; // increment object
+    if (Lt.preferences.subAnnual == false) {
```

Comment:
this logic is confusing.
If lt.preferences.subAnnual is false, but this.hasLateWood is true, this.subAnnual will be true. But then at 2846, if sub annual is false, we'll set this.subAnnual back to false. What's the desired outcome?

Reply:
Outcome is for subAnnual to be the 'boss'. If the user sets subAnnual to false, I want that to take precedent over hasLatewood. Although thinking about it now, if the user changes subAnnual, they'd have to delete all points and start over which would take care of assets which used hasLatewood. I see what you mean

Reply:
but then isn't this line the same as 
this.subAnnaul = Lt.preferences.subAnnual?

Reply:
In its current form no b/c this.hasLatewood is a diff object than this.subAnnual.

I'm going to consolidate this so there is only this.subAnnual.